feature/orchestrator/src/main/AndroidManifest.xml

```diff
@@ -22,9 +21,9 @@
             <intent-filter>
                 <action android:name="com.simprints.simodkadapter.REGISTER" />
                 <action android:name="com.simprints.simodkadapter.IDENTIFY" />
-                <action android:name="com.simprints.simodkadapter.UPDATE" />
                 <action android:name="com.simprints.simodkadapter.VERIFY" />
                 <action android:name="com.simprints.simodkadapter.CONFIRM_IDENTITY" />
+                <action android:name="com.simprints.simodkadapter.REGISTER_LAST_BIOMETRICS" />
```

Comment:
Oh, wow, we missed register last in most intent filters!?

Reply:
It looks like it wasn't implemented for some integration before refactoring at all.